Quick Start Guide and CLI installation guide updates for the Cloud CLI (#2119)
strapi/documentation: docusaurus/docs/dev-docs/installation/cli.md

- `--typescript`/`--ts`: Create a project in [TypeScript](/dev-docs/typescript).
- `--no-run`: Prevent Strapi from automatically starting the server (useful in combination with `--quickstart`).

> <!-- TODO: update with the flag to bypass the Cloud login -->

Author: - `--skip-cloud`: Automatically answers "Skip" to the Login/Signup question, which prevents the installation script from login into Strapi Cloud (useful in combination with `--quickstart`.

> <!-- TODO: update with the flag to bypass the Cloud login -->

Author: ```suggestion
- `--skip-cloud`: Automatically answers "Skip" to the Login/Signup question, which prevents the installation script from login into Strapi Cloud (useful in combination with `--quickstart`.
```

Suggested small update but subject to validation since I don't have any specs or visibility on the actual implementation of this 🤷  (@nathan-pichon , @derrickmehaffy)

Reviewer: lgtm 👍 
@gonbaum is currently implementing this flag 🙂

Reviewer 2: lgtm

Please use these experimental builds at your own risk. It is not recommended to use them in production.
:::

### Self-hosting your Strapi project

> When the installation script runs, the terminal will first ask you if you want to login/signup. Choosing `Login/signup` will create a free, 14-day trial [Strapi Cloud](/cloud/intro#what-is-strapi-cloud) account.

Author: ```suggestion
When the installation script runs, the terminal will first ask you if you want to login/signup. Choosing `Login/signup` will create a free, 14-day trial [Strapi Cloud](/cloud/intro#what-is-strapi-cloud) account as described in the [Quick Start Guide](/dev-docs/quick-start).
```

> When the installation script runs, the terminal will first ask you if you want to login/signup. Choosing `Login/signup` will create a free, 14-day trial [Strapi Cloud](/cloud/intro#what-is-strapi-cloud) account as described in the [Quick Start Guide](/dev-docs/quick-start).

Reviewer: I think the wording is misleading.
Its not a 14-day trial "Cloud account", but a 14-day trial Project in a Cloud account.

> When the installation script runs, the terminal will first ask you if you want to login/signup. Choosing `Login/signup` will create a free, 14-day trial [Strapi Cloud](/cloud/intro#what-is-strapi-cloud) account as described in the [Quick Start Guide](/dev-docs/quick-start).

Author: ```suggestion
When the installation script runs, the terminal will first ask you if you want to login/signup. Choosing `Login/signup` will create a free, 14-day trial [Strapi Cloud](/cloud/intro#what-is-strapi-cloud) project as described in the [Quick Start Guide](/dev-docs/quick-start).
```

> If you prefer hosting the project by yourself, use the arrow keys to select `Skip`. The script will resume and create a local project. To deploy this project and host it online, you will then have to host the project's code on a repository (e.g., on GitHub) before following a [3rd-party deployment guide](/dev-docs/deployment).

Reviewer: Users can still use `strapi login` and `strapi deploy` if they skip the cloud part in the create-strapi-app.

> If you prefer hosting the project by yourself, use the arrow keys to select `Skip`. The script will resume and create a local project. To deploy this project and host it online, you will then have to host the project's code on a repository (e.g., on GitHub) before following a [3rd-party deployment guide](/dev-docs/deployment).

Author: ```suggestion
If you prefer skipping this Strapi Cloud login part, use the arrow keys to select `Skip`. The script will resume and create a local project. To deploy this project and host it online, you could later choose to:
- host it yourself by pushing the project's code to a repository (e.g., on GitHub) before following a [3rd-party deployment guide](/dev-docs/deployment),
- or use the [Cloud CLI](/cloud/cli/cloud-cli) commands to login to Strapi Cloud and deploy your project there.
```

Author: Is it better like this, @nathan-pichon? 😊

Reviewer: Yes absolutely

If you're not already familiar with GitHub, the togglable content below should get you started 👇

<details>
<summary>Steps required to push your Strapi project code to GitHub:</summary>

1. In the terminal, ensure you are still in the folder that hosts the Strapi project you created.

> 2. Run the `git init` command to initialize git for this folder.

Author: Not a big deal, though, IMHO. This part is mostly for people who absolutely don't know how to do it. My point is more that if they don't do it, subsequent git commands will not work, whereas if they just reset their history it's fine at this point since at step 4 we just create an "initial commit".

3. Run the `git add .` command to add all modified files to the git index.
4. Run the `git commit -m "Initial commit"` command to create a commit with all the added changes.

> 5. Log in into your GitHub account and [create a new repository](https://docs.github.com/en/repositories/creating-and-managing-repositories/quickstart-for-repositories). Give the new repository a name, for instance `my-first-strapi-project`, and remember this name.

Reviewer 3: ```suggestion
5. Log in to your GitHub account and [create a new repository](https://docs.github.com/en/repositories/creating-and-managing-repositories/quickstart-for-repositories). Give the new repository a name, for instance `my-first-strapi-project`, and remember this name.
```

6. Go back to the terminal and push your local repository to GitHub:

>   a. Run a command similar to the following: `git remote add origin git@github.com:yourname/my-first-strapi-project.git`, ensuring you replace `yourname` by your actual GitHub profile name, and `my-first-strapi-project` by the actual name you used at step 4.

Reviewer 3: ```suggestion
  a. Run a command similar to the following: `git remote add origin git@github.com:yourname/my-first-strapi-project.git`, ensuring you replace `yourname` by your own GitHub profile name, and `my-first-strapi-project` by the actual name you used at step 4.
```

Please use these experimental builds at your own risk. It is not recommended to use them in production.
:::

> ### Self-hosting your Strapi project

Author: ```suggestion
### Skipping the Strapi Cloud login step
```

When the installation script runs, the terminal will first ask you if you want to login/signup. Choosing `Login/signup` will create a free, 14-day trial [Strapi Cloud](/cloud/intro#what-is-strapi-cloud) project as described in the [Quick Start Guide](/dev-docs/quick-start).

If you prefer skipping this Strapi Cloud login part, use the arrow keys to select `Skip`. The script will resume and create a local project. To deploy this project and host it online, you could later choose to:
- host it yourself by pushing the project's code to a repository (e.g., on GitHub) before following a [3rd-party deployment guide](/dev-docs/deployment),
- or use the [Cloud CLI](/cloud/cli/cloud-cli) commands to login to Strapi Cloud and deploy your project there.

> If you want to host your project yourself and are not already familiar with GitHub, the togglable content below should get you started👇.

Author: ```suggestion
If you want to host your project yourself and are not already familiar with GitHub, the following togglable content  should get you started👇.
```

> If you want to host your project yourself and are not already familiar with GitHub, the following togglable content  should get you started👇.

Author: ```suggestion
If you want to host your project yourself and are not already familiar with GitHub, the following togglable content should get you started👇.
```
